apps/wallet/src/ui/app/components/accounts/AccountBalanceItem.tsx

```diff
@@ -112,7 +109,40 @@ export function AccountBalanceItem({
 
     const hasAccountAssets = !!ownedObjects?.pages?.[0]?.[0]?.data?.length;
 
-    const hasVestingObjects = !!vestingObjects?.pages?.[0]?.[0]?.data?.length;
+    const {
+        data: vestingObjects,
+        hasNextPage,
+        fetchNextPage,
+    } = useGetOwnedObjectsMultipleAddresses(
+        addresses,
+        {
+            MatchAny: [{ StructType: TIMELOCK_IOTA_TYPE }, { StructType: TIMELOCK_STAKED_TYPE }],
+        },
+        OBJECT_PER_REQ,
```

Comment:
If we are going to search in some timelocked objects it would be good to search for more than 1